### ISSUE TITLE
Inconsistency in `lint/a11y/noSvgWithoutTitle` Lint Rule for aria-label and aria-labelledby

### ISSUE THREAD

**Issue by k-s-h-r** [biomejs/biome#784]
### Environment information

```bash
Version: 1.3.3
```

### What happened?

I've encountered an issue with the lint/a11y/noSvgWithoutTitle lint rule, where the behavior of aria-label is incorrectly matching that of aria-labelledby. While aria-label can be used independently, aria-labelledby requires specifying an ID.

#### Error Occurs

```html
<svg
    role="img"
    aria-label="sample"
>
    <span id="title">pass</span>
</svg>
```

#### No Error Occurs

```html
<svg
    role="img"
    aria-label="title"
>
    <span id="title">pass</span>
</svg>
```

#### No Error Should Occur with aria-label Specified Alone

```html
<svg
    role="img"
    aria-label="title"
>
    <span>pass</span>
</svg>
```

### Rule name

lint/a11y/noSvgWithoutTitle

### Playground link

https://biomejs.dev/playground?code=PABkAGkAdgA%2BAAoAIAAgADwAaAAyAD4AQQBjAHQAdQBhAGwAbAB5ACAAdgBhAGwAaQBkADwALwBoADIAPgAKACAAIAAKACAAIAA8AHMAdgBnACAAcgBvAGwAZQA9ACIAaQBtAGcAIgAgAGEAcgBpAGEALQBsAGEAYgBlAGwAPQAiAHMAYQBtAHAAbABlACIAPgAKACAAIAAgACAAPABzAHAAYQBuAD4AcABhAHMAcwA8AC8AcwBwAGEAbgA%2BAAoAIAAgADwALwBzAHYAZwA%2BAAoAIAAgAAoAIAAgADwAcwB2AGcACgAgACAAIAAgAHIAbwBsAGUAPQAiAGkAbQBnACIACgAgACAAIAAgAGEAcgBpAGEALQBsAGEAYgBlAGwAbABlAGQAYgB5AD0AIgB0AGkAdABsAGUAIgAKACAAIAA%2BAAoAIAAgACAAIAAgACAAPABzAHAAYQBuACAAaQBkAD0AIgB0AGkAdABsAGUAIgA%2BAHAAYQBzAHMAPAAvAHMAcABhAG4APgAKACAAIAA8AC8AcwB2AGcAPgAKAAoAIAAgADwAcwBwAGEAbgA%2BAC0ALQAtAC0ALQAtAC0ALQAtAC0ALQAtAC0ALQAtAC0ALQAtAC0ALQAtAC0ALQAtADwALwBzAHAAYQBuAD4ACgAgACAACgAgACAAPABoADIAPgBBAGMAdAB1AGEAbABsAHkAIABpAG4AdgBhAGwAaQBkADwALwBoADIAPgAKACAAIAA8AHAAPgBUAGgAZQAgAGwAYQBiAGUAbAAgAHMAaABvAHUAbABkACAAYgBlACAAJwBwAGEAcwBzACcAIABiAHUAdAAgAGkAdAAgAGIAZQBjAG8AbQBlAHMAIAAnAHQAaQB0AGwAZQAnAC4APAAvAHAAPgAKACAAIAAKACAAIAA8AHMAdgBnAAoAIAAgACAAIAByAG8AbABlAD0AIgBpAG0AZwAiAAoAIAAgACAAIABhAHIAaQBhAC0AbABhAGIAZQBsAD0AIgB0AGkAdABsAGUAIgAKACAAIAA%2BAAoAIAAgACAAIAA8AHMAcABhAG4AIABpAGQAPQAiAHQAaQB0AGwAZQAiAD4AcABhAHMAcwA8AC8AcwBwAGEAbgA%2BAAoAIAAgADwALwBzAHYAZwA%2BAAoACgAgACAACgA8AC8AZABpAHYAPgA%3D

### Expected result

The aria-label can be used on its own.
An error should not occur when only aria-label is used.

### Code of Conduct

- [X] I agree to follow Biome's Code of Conduct

**Comment by unvalley** [comment]
@k-s-h-r Thank you for reporting, I'm working on #788
